CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## 1.6.3 (not yet released)
 * ADDED: Detect and report on damaged pastes (#1218)
-* CHANGED: Upgrading libraries to: zlib 1.3
+* CHANGED: Increase zlib compression level from 7 to 9
```

Comment:
Is there a particular reason to do so? Just curious hmm… Whether that brings any effect worth it.

(BTW if one wanted, we could make it configurable, but yeah… likely not worth the effort.)

Reply:
Not any reason in particular. The default still is 6 and I just noticed that we had it set to 7. I think that we could just as well increase it to the max, since CPUs tend to be more powerful over time. Admittedly the gains will likely be minimal. For example, [this benchmark](https://www.rootusers.com/gzip-vs-bzip2-vs-xz-performance-comparison/) suggests that changing compression from  7 to 9 taking about 1.8 times as long, for a 0.1% gain on a 21% compression ratio. Our data tends to be relatively small, I'd guess log file samples to compress pretty well.

Worth keeping it at 9, or better to stay on the old default? Opinions?

Reply:
Dunno without any reason and when seeing these stats I would keep it as it is.

Reply:
Okay, so good I asked. Thanks!